src/azul/service/hca_response_v5.py

```diff
@@ -509,17 +530,19 @@ def map_entries(self, entry):
             'fileTypeSummaries': [FileTypeSummary.for_aggregate(aggregate_file).to_json()
                                   for aggregate_file in entry["contents"]["files"]]
         }
-        return HitEntry(protocols=self.make_protocols(entry),
-                        entryId=entry["entity_id"],
-                        sources=self.make_sources(entry),
-                        projects=self.make_projects(entry),
-                        samples=self.make_samples(entry),
-                        specimens=self.make_specimens(entry),
-                        cellLines=self.make_cell_lines(entry),
-                        donorOrganisms=self.make_donors(entry),
-                        organoids=self.make_organoids(entry),
-                        cellSuspensions=self.make_cell_suspensions(entry),
-                        **kwargs)
+        # By calling the JsonObject constructor with a dictionary instead of
+        # keyword arguments, we avoid an unnecessary deepcopy of the values.
+        return HitEntry(dict(protocols=self.make_protocols(entry),
```

Comment:
We should do the wrapping in the AzulJsonObject constructor. It should only accept kwargs and always wrap them. That way we collect all the quirks in a single place.